force-app/main/default/classes/JobApplicationTriggerHandler.cls

```diff
@@ -154,4 +166,29 @@ public with sharing class JobApplicationTriggerHandler {
     private static Boolean hasContactLinkedToCompanyRef(Id companyIdFromJobApp, Map<Id, Id> accountIdToContactIdMap) {
         return accountIdToContactIdMap.get(companyIdFromJobApp) != null;
     }
+
+        // Calculate Take Home Pay
+    public static void estimateTakeHomePay(List<JobApplication__c> triggerDotNew) {
+        List<JobApplication__c> jobAppsWithSalaries = new List<JobApplication__c>();
+        for(JobApplication__c jobApp : triggerDotNew) {
+            if(jobApp.Salary__c != null) {
+                calculatePay(jobApp);
+                jobAppsWithSalaries.add(jobApp);
+            }
+        }
+    }
+
+    public static void calculatePay(JobApplication__c jobApp) {
```

Comment:
```suggestion
    private static void calculatePay(JobApplication__c jobApp) {
```

Reply:
As this appears to be method only used within this class and not called from another I feel this should be labeled as private